etc/adminhtml/system/adyen_boleto.xml

```diff
@@ -50,8 +50,7 @@
             <config_path>payment/adyen_boleto/boletotypes</config_path>
         </field>
         <field id="delivery_days" translate="label" type="text" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="0">
-            <label>Days for Delivery</label>
-            <tooltip>How many days to be added to the current date for delivery. Only numbers allowed.</tooltip>
+            <label>Days for delivery and payment</label>
             <frontend_class>validate-number</frontend_class>
             <config_path>payment/adyen_boleto/delivery_days</config_path>
         </field>
```

Comment:
can you please add in "specificcountry" the dependency on "allowspecific"(lines 69-70)?
like we do here:
https://github.com/Adyen/adyen-magento2/blob/b5cb60d258810ae41dd714592a0cd83519cc9021/etc/adminhtml/system/adyen_hpp.xml#L82-L84
so we show one less field if "allowspecific" is disabled